Xamarin.PropertyEditing.Mac/Controls/EntryPropertyEditor.cs

```diff
@@ -2,9 +2,34 @@
 using AppKit;
 using Foundation;
 using Xamarin.PropertyEditing.ViewModels;
+using ObjCRuntime;
 
 namespace Xamarin.PropertyEditing.Mac
 {
+	class TextNextResponderDelegate : NSTextFieldDelegate
+	{
+		public RowProxyResponder ProxyResponder { get; set; }
+
+		public override bool DoCommandBySelector (NSControl control, NSTextView textView, Selector commandSelector)
+		{
+			switch (commandSelector.Name) {
+			case "insertTab:":
+				if (ProxyResponder?.NextResponder () ?? false)
```

Comment:
You can just guarantee the ProxyResponder is not null and that will make the code readable.

Reply:
In example, pass it in via constructor and do `responder ?? throw new ArgumentNullException()`, that way you don't have to null check everywhere.